src/common/CMakeLists.txt

```diff
@@ -30,7 +30,14 @@ endif()
 if(${OQS_USE_SHA2_OPENSSL})
     set(SHA2_IMPL sha2/sha2_ossl.c)
 else()
-    set(SHA2_IMPL sha2/sha2_c.c)
+    set(SHA2_IMPL sha2/sha2.c sha2/sha2_c.c)
+    if (OQS_DIST_ARM64v8_BUILD)
+       set(SHA2_IMPL ${SHA2_IMPL} sha2/sha2_ni.c)
+       set_source_files_properties(sha2/sha2_ni.c PROPERTIES COMPILE_FLAGS -mcpu=cortex-a53+crypto)
+    elseif (OQS_USE_ARM_SHA2_INSTRUCTIONS)
+       # Assume if we are here we are comopiling native
```

Comment:
Typo in comment. But I'm wondering more what/who sets `OQS_USE_ARM_SHA2_INSTRUCTIONS`? Or should this go into the build instructions Wiki?

Reply:
`OQS_USE_ARM_SHA2_INSTRUCTIONS` is set by CMake in `.CMake/detect_gcc_clang_intrinsics.c`.

Reply:
Thank you for catching that! I just updated it.